db/migrate/20191220023114_add_game_id_to_piece.rb

```diff
@@ -1,5 +1,4 @@
 class AddGameIdToPiece < ActiveRecord::Migration[5.2]
   def change
-    add_index :pieces, :game_id
```

Comment:
I should of put a comment in but this was giving me an error when I ran `rake db:migrate` something about it already existing. So I had to change this file to allow the migration to continue.

Reply:
Michele has the same issue. Not sure why. She was able to fix it by running `rake db:shcema:load` without removing the migration history. I think this will be fine though.